src/openrct2/world/TileElement.h

```diff
@@ -189,7 +189,7 @@ struct PathElement : TileElementBase
     union
     {
         uint8_t additionStatus; // 7
-        ride_id_t rideIndex;
+        uint8_t rideIndex;
```

Comment:
This looks weird, it's the only place you let go of the `ride_id_t` alias, even though. Why?

Reply:
Changes to save import / export are needed before we can change the tile element structs, so the ride id in tile elements has to remain 8-bit until we make those further changes.